oci-register-machine.go

```diff
@@ -79,6 +79,10 @@ func TerminateMachine(name string) error {
 			return err
 		}
 	}
+	if len(name) > 32 {
```

Comment:
This is problematic as it could lead to clashes. I think we might have to use the ID here.